src/Features/CSharp/Portable/SignatureHelp/ConstructorInitializerSignatureHelpProvider.cs

```diff
@@ -82,8 +82,11 @@ private static bool IsArgumentListToken(ConstructorInitializerSyntax expression,
                 return null;
             }
 
+            var currentCunstructor = semanticModel.GetDeclaredSymbol(( constructorInitializer.Parent as ConstructorDeclarationSyntax ));
```

Comment:
No parens needed.
No surrounding spaces needed.
Should just be a straight case as all ConstructorInitializers must be parented by a ConstructorDeclaration.